xml/libvirt_host.xml

```diff
@@ -0,0 +1,1997 @@
+      specified domain, run:
+     </para>
+<screen>&prompt.sudo;virsh domiflist nated_sles12sp3
+Interface  Type       Source       Model       MAC
+---------------------------------------------------------
+vnet0      network    vnet_nated   virtio      52:54:00:9e:0d:2b</screen>
+    </sect4>
+    <sect4 xml:id="libvirt-networks-virtual-virsh-info">
+     <title>Getting Details about a Network</title>
+     <para>
+      To get detailed information about a network, run:
+     </para>
+<screen>&prompt.sudo;virsh net-info vnet_routed
+Name:           vnet_routed
+UUID:           756b48ff-d0c6-4c0a-804c-86c4c832a498
+Active:         yes
+Persistent:     yes
+Autostart:      yes
+Bridge:         virbr5</screen>
+    </sect4>
+    <sect4 xml:id="libvirt-networks-virtual-virsh-start">
+     <title>Starting a Network</title>
+     <para>
+      To start an inactive network that was already defined, find its name (or
+      unique identifier, UUID) with:
+     </para>
+<screen>&prompt.sudo;virsh net-list --inactive
+ Name                 State      Autostart     Persistent
+----------------------------------------------------------
+ vnet_isolated        inactive   yes           yes</screen>
+     <para>
+      Then run:
+     </para>
+<screen>&prompt.sudo;virsh net-start vnet_isolated
+Network vnet_isolated started</screen>
+    </sect4>
+    <sect4 xml:id="libvirt-networks-virtual-virsh-stop">
+     <title>Stopping a Network</title>
+     <para>
+      To stop an active network, find its name (or unique identifier, UUID)
+      with:
+     </para>
+<screen>&prompt.sudo;virsh net-list --inactive
+ Name                 State      Autostart     Persistent
+----------------------------------------------------------
+ vnet_isolated        active     yes           yes</screen>
+     <para>
+      Then run:
+     </para>
+<screen>&prompt.sudo;virsh net-destroy vnet_isolated
+Network vnet_isolated destroyed</screen>
+    </sect4>
+    <sect4 xml:id="libvirt-networks-virtual-virsh-undefine">
+     <title>Removing a Network</title>
+     <para>
+      To remove the definition of an inactive network from &vmhost;
+      permanently, run:
+     </para>
+<screen>&prompt.sudo;virsh net-undefine vnet_isolated
+Network vnet_isolated has been undefined</screen>
+    </sect4>
+   </sect3>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="libvirt-host-storage">
+  <title>Configuring a Storage Pool</title>
+
+  <para>
+   When managing a &vmguest; on the &vmhost; itself, you can access the
+   complete file system of the &vmhost; to attach or create virtual hard disks
+   or to attach existing images to the &vmguest;. However, this is not possible
+   when managing &vmguest;s from a remote host. For this reason, &libvirt;
+   supports so called <quote>Storage Pools</quote>, which can be accessed from
+   remote machines.
+  </para>
+
+  <tip>
+   <title>CD/DVD ISO images</title>
+   <para>
+    To be able to access CD/DVD ISO images on the &vmhost; from remote, they
```

Comment:
From remote what? From remote hosts, or remote clients, or remote machines in general?